commands/configure/index.js

```diff
@@ -92,17 +92,34 @@ const setSkippedExperience = (api, application) => {
   return api.application.patch({ applicationId: application.applicationId, application: { ftueTracking: application.ftueTracking } });
 };
 
+const getApiURL = async (userConfig) => {
+  const keys = Object.keys(userConfig);
+  if (keys.length < 1) {
+    logError('User Configuration file missing, run losant login to generate this file.');
```

Comment:
you are calling loadUserConfig right before you call this, and loadUserConfig will throw this error if the config is empty. It should be impossible to get to this error.